vk.py

```diff
@@ -1,13 +1,15 @@
+"""Module for work with VK api."""
 import os
 import json
 import urllib.request
-import constants
 
 
-def vk_get():
-    # get data from vk.com
-    vk_id = constants.vk_id
-    adr = 'https://api.vk.com/method/wall.get?owner_id={0}&count=1'.format(vk_id)
+def vk_get(vk_id):
+    """Get data from vk.com."""
+    adr = (
+        "https://api.vk.com/method" +
```

Comment:
BTW, what do you think if you move links to variables? Like `API_DOMAIN` in each file